synapse/storage/state.py

```diff
@@ -448,6 +448,7 @@ def get_current_state_ids(self, room_id):
         Returns:
             deferred: dict of (type, state_key) -> event_id
         """
+
```

Comment:
please try to avoid unrelated reformatting in your PRs; it makes git history harder to read.